.gitignore

```diff
@@ -2,7 +2,7 @@
 .gitignore
 
 # Ignore configuration files that may contain sensitive information.
-/settings*.php
+# /settings*.php
```

Comment:
Why this?

Reply:
I had to do that in .gitignore to commit settings.php but I don't remember committing the change to .gitignore. It should be ignored by its own definition.